src/components/ui/MarkdownViewer/index.tsx

```
@@ -3,6 +3,24 @@ import ReactMarkdown from "react-markdown";
 import remarkBreaks from "remark-breaks";
 import remarkGfm from "remark-gfm";
 import { Image } from "./Image";
+import { YouTube } from "@next/third-parties/google";
```

ESLint

[error] 'YouTube' is defined but never used.

(@typescript-eslint/no-unused-vars)